results.html

```diff
@@ -2,42 +2,201 @@
-    />
-    <!-- Project Stylesheets -->
-    <link rel="stylesheet" href="./assets/css/reset.css" />
-    <link rel="stylesheet" href="./assets/css/index.css" />
-    <!--Let browser know website is optimized for mobile-->
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  </head>
-
-  <body>
+
+<head>
+  <!--Import Google Icon Font-->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+  <!--Import materialize.css-->
+  <!-- <link type="text/css" rel="stylesheet" href="css/materialize.min.css" media="screen,projection" /> -->
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Pridi:wght@200;300;400&family=Source+Sans+Pro:ital,wght@0,200;0,300;1,200;1,300&family=Tinos:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+    rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" />
+
+  <!-- Framework Stylesheets -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" />
+  <!-- Project Stylesheets -->
+  <link rel="stylesheet" href="./assets/css/reset.css" />
+  <link rel="stylesheet" href="./assets/css/index.css" />
+  <!--Let browser know website is optimized for mobile-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body class="hero-image"></body>>
+
+<main>
+  <div class="rowOne">
+    <div class="row">
+      <div class="col s5 m5 l5">
+        <img class="logo" src="./assets/images/corporate-logo-maker-with-icon-1517d (10).png" />
+      </div>
+      <div class="col s7">
+        <h1 id="cityName">City Name</h1>
+      </div>
+    </div>
+  </div>
+
+  <div class="rowTwo">
+    <div class="row"></div>
+    <div class="col">
+      <ul class="collapsible">
+        <li>
+          <div class="collapsible-header white-text opacityLi"><i class="large material-icons">search</i><input
+              type="text" class="search-bar" /></div>
+          <div class="collapsible-body">
+            <div class="search-container">
+
+              <div class="search-results container">
+                <div class="container-item">item1</div>
+                <div class="container-item">item2</div>
+                <div class="container-item">item3</div>
+              </div>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div class="col">
+      <ul class="collapsible">
+        <li>
+          <div class="collapsible-header white-text opacityLi"><i class="large material-icons">view_list</i>City
+            Information</div>
+          <div class="collapsible-body">
+            <div id="CityData" class="card">
```

Comment:
```
            <div
              **id="Demographics"**
              class="card col s12 m6"
              style="height: 350px"
            >
              <h2>City Name</h2>
              <p>Demographic Data: 1</p>
              <p>Demographic Data: 2</p>
              <p>Demographic Data: 3</p>
              <p>Demographic Data: 4</p>
              <p>Demographic Data: 5</p>
            </div>
            <div class="card col s12 m6" style="height: 350px">
              **<div id="CityMap"></div>**
            </div>
```
These ID attributes are crucial for the JS to work.  We don't need an `<iframe>`, just an empty `<div>` with the id "CityMap"